src/main/scala/com/cognite/spark/datasource/test.sc

```diff
@@ -1,2 +0,0 @@
-val a = 10
-a + 10
```

Comment:
Weird. Where did this come from?

Reply:
Looks like it snuck in here https://github.com/cognitedata/cdp-spark-datasource/commit/740e3672ac9f473048f00ef638520aa40f87b304#diff-fe8c54b29b54518d08cd9ea7e35a63b6